Python/aot_ceval_jit.c

```diff
@@ -3696,7 +3709,13 @@ void* jit_func(PyCodeObject* co, PyThreadState* tstate) {
                 deferred_vs_apply(Dst);
             }
             emit_store64_mem(Dst, vsp_idx, f_idx, offsetof(PyFrameObject, f_stacktop));
-            | branch ->exit_yielding
+            if (exit_yielding_label) {
+                | branch ->exit_yielding
```

Comment:
I guess it's not necessary with the current code, but I would feel more comfortable if this set `exit_yielding_label_used=1` here